sycl/test-e2e/Basic/kernel_max_wg_size.cpp

```diff
@@ -0,0 +1,54 @@
+// RUN: %{build} -o %t.out
+// RUN: %{run} %t.out
+
+// REQUIRES: gpu && gpu-intel-pvc
```

Comment:
You are right, this can be enabled for both DG2 and PVC. grf_size 256 (large) can be specified only for pvc and dg2 irght now according to https://github.com/intel/llvm/blob/sycl/sycl/doc/extensions/experimental/sycl_ext_intel_grf_size.asciidoc, so it doesn't seem feasible to enable this test for other hardware.